src/OrchardCore/OrchardCore.Users.Abstractions/Services/IUserTimeZoneService.cs

```diff
@@ -0,0 +1,13 @@
+using System.Threading.Tasks;
+using OrchardCore.Modules;
+
+namespace OrchardCore.Users.Services;
+
+public interface IUserTimeZoneService
```

Comment:
I looked at the code and I think it was on purpose that we didn't create an interface. This service is specific to the timezone resolver implementation. There is no reason to replace this implementation with another one. One would just provide a different selector with a different priority.

Reply:
Ok then we can remove. I agree.